### PR TITLE
APIGOV-17009 Update page in Connect API Manager “gateway-administation”

### PR DESCRIPTION
**Pull request by dfeldick**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by dfeldick** [comment]
@kflan-io, @jcollins-axway @sbolosan please review

File: content/en/docs/central/connect-api-manager/gateway-administation.md [hunk 1]
```diff
@@ -587,7 +587,7 @@ cd /home/APIC-agents
 2. Start the Docker Discovery Agent pointing to the `env_vars` file and the keys directory. `pwd` relates to the local directory where the docker command is run. For Windows, the absolute path is preferred.
 
    ```shell
-   docker run --env-file ./env_vars -v <pwd>/keys:/keys  axway-docker-public-registry.bintray.io/agent/v7-discovery-agent:latest
+   docker run --env-file ./env_vars -v <pwd>/keys:/keys axway.jfrog.io/ampc-public-docker-release/agent/aws-apigw-discovery-agent:latest
    ```
```
Comment (kflan-io) [inline content/en/docs/central/connect-api-manager/gateway-administation.md:591]: You put AWS...
edit to v7:
docker run --env-file ./env_vars -v <pwd>/keys:/keys axway.jfrog.io/ampc-public-docker-release/agent/v7-discovery-agent:latest

File: content/en/docs/central/connect-api-manager/gateway-administation.md [hunk 2]
```diff
@@ -1178,7 +1178,7 @@ cd /home/APIC-agents
 2. Start the Traceability Agent pointing to the `env_vars` file, `keys`, and the logging `events` directory. `pwd` relates to the local directory where the docker command is run. For Windows, the absolute path is preferred.
 
    ```shell
-   docker run --env-file ./env_vars -v <pwd>/keys:/keys -v <pwd>/events:/events axway-docker-public-registry.bintray.io/agent/v7-traceability-agent:latest
+   docker run --env-file ./env_vars -v <pwd>/keys:/keys -v <pwd>/events:/events axway.jfrog.io/ampc-public-docker-release/agent/aws-apigw-traceability-agent:latest
    ```
```
Comment (kflan-io) [inline content/en/docs/central/connect-api-manager/gateway-administation.md:1182]: Same

**Review by kflan-io — CHANGES_REQUESTED** [review]
you put AWS and need to edit back to v7

**Review by cbordier — APPROVED** [review]
I approve the changes

**Review by kflan-io — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**